diff --git a/.nuget/NuGet.targets b/.nuget/NuGet.targets
index 1bd2eee..db7b54c 100644
--- a/.nuget/NuGet.targets
+++ b/.nuget/NuGet.targets
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">$(MSBuildProjectDirectory)\..\</SolutionDir>
         <NuGetToolsPath>$([System.IO.Path]::Combine($(SolutionDir), ".nuget"))</NuGetToolsPath>
-        <NuGetExePath>$(NuGetToolsPath)\nuget.exe</NuGetExePath>
+        <NuGetExePath>nuget</NuGetExePath>
         <PackagesConfig>$([System.IO.Path]::Combine($(ProjectDir), "packages.config"))</PackagesConfig>
         <PackagesDir>$([System.IO.Path]::Combine($(SolutionDir), "packages"))</PackagesDir>
         <PackageOutputDir Condition="$(PackageOutputDir) == ''">$(TargetDir.Trim('\\'))</PackageOutputDir>
@@ -51,4 +51,4 @@
               LogStandardErrorAsError="true"
               WorkingDirectory="$(NuGetToolsPath)" />
     </Target>
-</Project>
\ No newline at end of file
+</Project>
