diff --git a/CMakeLists.txt b/CMakeLists.txt
index f57a733a..c2acaf65 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,7 +41,7 @@ set_package_properties(FFmpeg PROPERTIES DESCRIPTION "Video Tag reader" URL "htt
 find_package(EPub)
 set_package_properties(EPub PROPERTIES DESCRIPTION "Ebook epub reader" URL "http://sourceforge.net/projects/ebook-tools" TYPE OPTIONAL PURPOSE "Support for epub metadata")
 
-find_package(QMobipocket)
+find_package(QMobipocket PATHS /usr/kde4 NO_DEFAULT_PATH)
 set_package_properties(QMobipocket PROPERTIES DESCRIPTION "Mobipocket epub reader" URL "https://projects.kde.org/projects/kde/kdegraphics/kdegraphics-mobipocket" TYPE OPTIONAL PURPOSE "Support for mobi metadata")
 
 find_package(KFileMetaData)
